### ISSUE TITLE
All examples `panic!` with `NoAvailableAdapter` and return the same error - Void Linux

### ISSUE THREAD

**Issue by mattheusroxas** [nannou-org/nannou#724]
### Error given when examples are run (with backtrace)
```
thread 'main' panicked at 'internal error: entered unreachable code: Unexpected result - driver bug? Err(ERROR_UNKNOWN)', /home/mattheus/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/gfx-backend-vulkan-0.5.11/src/device.rs:2091:18
stack backtrace:
   0: rust_begin_unwind
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/panicking.rs:493:5
   1: core::panicking::panic_fmt
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/core/src/panicking.rs:92:14
   2: gfx_backend_vulkan::device::<impl gfx_hal::device::Device<gfx_backend_vulkan::Backend> for gfx_backend_vulkan::Device>::create_swapchain
   3: <gfx_backend_vulkan::window::Surface as gfx_hal::window::PresentationSurface<gfx_backend_vulkan::Backend>>::configure_swapchain
   4: wgpu_core::device::<impl wgpu_core::hub::Global<G>>::device_create_swap_chain
   5: nannou::window::Builder::build
   6: nannou::app::Builder<M,E>::run
   7: art::main
```
### System info
* **OS:** Void Linux
* **Vulkan drivers**: `vulkan-loader`, `mesa-vulkan-radeon` and `amdvlk`. Which are all the ones required to be installed, per the documentation; [Driver section, Void Linux docs](https://docs.voidlinux.org/config/graphical-session/graphics-drivers/amd.html)

### WGPU
I've already tested out a lot of WGPU's examples (on both the `master` and `v0.6` branches); `cube`, `water`, `boids` `hello-triangle` and shadow, and it's worked out fine.

**Comment by mitchmindtree** [comment]
Can you please share which examples you have tried to run?

If possible, could you try the following examples:

- `draw`
- `draw_texture`
- `simple_ui`
- `wgpu_triangle`
- `wgpu_triangle_raw_frame`

Could you also try without the `--release` flag and post the stack trace again? This should give a bit more info about exactly where the `panic!` occurs.

**Comment by mattheusroxas** [comment]
### All those examples listed return (with backtrace):
```
thread 'main' panicked at 'internal error: entered unreachable code: Unexpected result - driver bug? Err(ERROR_UNKNOWN)', /home/mattheus/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/gfx-backend-vulkan-0.6.5/src/device.rs:2405:18
stack backtrace:
   0: rust_begin_unwind
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/panicking.rs:493:5
   1: core::panicking::panic_fmt
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/core/src/panicking.rs:92:14
   2: gfx_backend_vulkan::device::<impl gfx_backend_vulkan::Device>::create_swapchain
             at /home/mattheus/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/gfx-backend-vulkan-0.6.5/src/device.rs:2405:18
   3: <gfx_backend_vulkan::window::Surface as gfx_hal::window::PresentationSurface<gfx_backend_vulkan::Backend>>::configure_swapchain
             at /home/mattheus/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/gfx-backend-vulkan-0.6.5/src/window.rs:447:35
   4: wgpu_core::device::<impl wgpu_core::hub::Global<G>>::device_create_swap_chain
             at /home/mattheus/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-core-0.6.5/src/device/mod.rs:3344:13
   5: <wgpu::backend::direct::Context as wgpu::Context>::device_create_swap_chain
             at /home/mattheus/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.6.2/src/backend/direct.rs:600:9
   6: wgpu::Device::create_swap_chain
             at /home/mattheus/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.6.2/src/lib.rs:1531:17
   7: nannou::window::SwapChainBuilder::build
             at ./nannou/src/window.rs:340:26
   8: nannou::window::Builder::build
             at ./nannou/src/window.rs:818:13
   9: nannou::app::Builder<M,E>::run
             at ./nannou/src/app.rs:476:29
  10: nannou::app::SketchBuilder<E>::run
             at ./nannou/src/app.rs:518:9
  11: draw::main
             at ./examples/draw/draw.rs:4:5
  12: core::ops::function::FnOnce::call_once
             at /home/mattheus/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/function.rs:227:5
```

As for the later suggestion:
```
thread 'main' panicked at 'internal error: entered unreachable code: Unexpected result - driver bug? Err(ERROR_UNKNOWN)', /home/mattheus/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/gfx-backend-vulkan-0.5.11/src/device.rs:2091:18
stack backtrace:
   0: rust_begin_unwind
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/std/src/panicking.rs:493:5
   1: core::panicking::panic_fmt
             at /rustc/2fd73fabe469357a12c2c974c140f67e7cdd76d0/library/core/src/panicking.rs:92:14
   2: gfx_backend_vulkan::device::<impl gfx_hal::device::Device<gfx_backend_vulkan::Backend> for gfx_backend_vulkan::Device>::create_swapchain
             at /home/mattheus/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/gfx-backend-vulkan-0.5.11/src/device.rs:2091:18
   3: <gfx_backend_vulkan::window::Surface as gfx_hal::window::PresentationSurface<gfx_backend_vulkan::Backend>>::configure_swapchain
             at /home/mattheus/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/gfx-backend-vulkan-0.5.11/src/window.rs:455:35
   4: wgpu_core::device::<impl wgpu_core::hub::Global<G>>::device_create_swap_chain
             at /home/mattheus/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-core-0.5.6/src/device/mod.rs:2056:13
   5: wgpu_device_create_swap_chain
             at /home/mattheus/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-native-0.5.1/src/device.rs:356:5
   6: wgpu::Device::create_swap_chain
             at /home/mattheus/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.5.2/src/lib.rs:1005:17
   7: nannou::window::SwapChainBuilder::build
             at /home/mattheus/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/nannou-0.15.0/src/window.rs:335:26
   8: nannou::window::Builder::build
             at /home/mattheus/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/nannou-0.15.0/src/window.rs:816:13
   9: nannou::app::Builder<M,E>::run
             at /home/mattheus/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/nannou-0.15.0/src/app.rs:454:29
  10: art::main
             at ./main.rs:22:5
  11: core::ops::function::FnOnce::call_once
             at /home/mattheus/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/function.rs:227:5
```

**Comment by mitchmindtree** [comment]
Closed via #725.
